bin/common.bash

```diff
@@ -31,6 +31,9 @@ config_path="${CK8S_CONFIG_PATH}/${prefix}-config"
 sops_config="${CK8S_CONFIG_PATH}/.sops.yaml"
 state_path="${CK8S_CONFIG_PATH}/.state"
 
+# Set the path to search for dynamic inventories
+export TERRAFORM_STATE_ROOT="${config_path}"
```

Comment:
Does this need documentation update?

Reply:
I don't think so. Previous behavior was surprising: it found the state of all clusters under `CK8S_CONFIG_PATH`, so essentially trying to build a mega cluster by combining SC and all WCs. If you happened to know about this variable you could avoid it but it isn't really documented.
Now it just does what you would expect.

Edit: Seems like it is actually used in a snippet without explanation. Will remove it from there. :slightly_smiling_face: